pkg/codegen/python/gen.go

```diff
@@ -298,6 +269,34 @@ func (mod *modContext) genInit(exports []string) string {
 		}
 	}
 
+	// If there are subpackages, import them with importlib.
+	if len(mod.children) > 0 {
```

Comment:
Should we do an up-front check to see if the length == 1 and only contains `config`, and avoid emitting any code in that case.

See https://github.com/pulumi-bot/pulumi-azuread/compare/cac6eb2..8a632b7#diff-16689a6998d278c7164a5a1c26081cb3R26-R32

```python
# Make subpackages available:
submodules = [
    'config',
]
for pkg in submodules:
    if pkg != 'config':
        importlib.import_module(f'{__name__}.{pkg}')
```